pages/newUser.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Create Account</title>
+        <link href="../styles/styles.css" rel="stylesheet">
+    </head>
+    <body>
+    <h2 id="center">Create a new account</h2>
+    <div id="modalContainer" class="">
```

Comment:
do not use empty class props, if needed add them later. You can delete it for now

```suggestion
    <div id="modalContainer">
```